src/main/kotlin/io/sip3/salto/ce/Bootstrap.kt

```diff
@@ -53,6 +56,9 @@ open class Bootstrap : AbstractBootstrap() {
         vertx.deployVerticle(AttributesHandler::class, config)
         vertx.deployVerticle(SipCallHandler::class, config, instances)
         vertx.deployVerticle(SipMessageHandler::class, config, instances)
+        vertx.deployVerticle(SipTransactionHandler::class, config, instances)
```

Comment:
Let's rename `SipMessagesHandler` to `SipMessageHandler` and `SipMessageHandler` to `SipPacketHandler`. 
Also let's rearrange. So we load verticles starting the end of the chain `Call`, `Options`, then `Transaction` and only then `Packet`.